src/Entity/LtiInstance.php

```diff
@@ -72,4 +88,9 @@ public function getLtiSecret(): string
     {
         return $this->ltiSecret;
     }
+
+    public function getCreatedAt(): DateTimeImmutable
+    {
+        return DateTimeImmutable::createFromMutable((new DateTime())->setTimestamp($this->createdAt));
```

Comment:
return DateTimeImmutable::createFromFormat($this->createdAt, 'U');

Reply:
For me this is not working :( Could you please double check?